sympy/parsing/sympy_parser.py

```diff
@@ -989,7 +989,15 @@ def parse_expr(s, local_dict=None, transformations=standard_transformations,
         raise TypeError('expecting local_dict to be a dict')
 
     if global_dict is None:
-        global_dict = {}
+        from sympy.core.power import Pow
+        from sympy.functions.elementary.complexes import Abs
+        from sympy.functions.elementary.miscellaneous import Max, Min
```

Comment:
Please move these imports to the top level (if possible, some may have to stay here to avoid cyclic imports).